app/services/scholars_archive/param_scrubber.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module ScholarsArchive
+  # checks peer reviewed
+  class ParamScrubber
+    def self.scrub(params, hash_key)
+      params[hash_key].each_pair do |attr, value|
+        if value.is_a?(Array)
+        # Set the mapped array on the params hash
+          set_values(params, hash_key, attr, mapped_values(value))
+        # If value is a Hash
+        elsif value.is_a? ActionController::Parameters
+          # Recursively dig into the hashes to find the raw value and set them on the params hash
+          set_values(params, hash_key, attr, extract_hash_values(value.to_hash).to_hash)
+        elsif value.is_a? String
+          # set the stripped string on the params 
+          set_params(params, hash_key, attr, value)
+        end
+      end
+    end
+
+    private
+
+    def self.extract_hash_values(hash)
+      hash.each_pair do |key, value|
```

Comment:
I'm surprised rubocop didn't warn on this, it seems like it would suggest

```
      hash.each_pair do |key, value|
        return strip_and_set(hash, key, value) if value.respond_to?(:strip)

        # recursively process nested hashes
        extract_hash_values(value)
      end
```